packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -20,32 +20,32 @@ class FlowHelper {
 
   getDatasetSite = pMemoize(async () => {
     return this.state.getDatasetSite();
-  });
+  }, { cachePromiseRejection: true });
```

Comment:
Very simple solution. Based on my understanding of the Test Suite, this shouldn't cause any issues and is in fact the behaviour that we want. Let me know what you think though as I'm still not too familiar with the project and may have overlooked something

Reply:
Yep this makes sense - we only want it to be called once and cached once within the flow